README.md

```diff
@@ -9,3 +9,78 @@ We welcome contributions from everyone. Just open an [issue](https://github.com/
+| `image.pullPolicy`                          | The image pull policy                                                                                 | `IfNotPresent`              |
+| `service.port`                              | The port that the service listens on                                                                  | `15432`                     |
+| `resources`                                 | Resource requests and limits for the container                                                        | `{}`                        |
+| `gatewaydPluginsConfig.enabled`             | Determines whether the `gatewayd_plugins.yaml` ConfigMap is mounted to the container. If enabled, a volume and volumeMount are added to the deployment. | `false`                     |
+| `nodeSelector`                              | Node selector for the pod                                                                             | `{}`                        |
+| `affinity`                                  | Affinity for the pod                                                                                  | `{}`                        |
+| `tolerations`                               | Tolerations for the pod                                                                               | `[]`                        |
+| `ingress.enabled`                           | Determines whether an Ingress resource should be created                                               | `false`                     |
+| `gatewayd.fullname`                         | The full name of the deployment, used as part of the Ingress resource name                             |                             |
+| `service.port`                              | The port that the service listens on, used as the service port in the Ingress rules                    |                             |
+| `ingress.className`                         | The Ingress class to assign to the Ingress resource. This is only used for Kubernetes versions less than 1.18.                                                     |                             |
+| `ingress.annotations`                       | Annotations to add to the Ingress resource. If `ingress.className` is set and the Kubernetes version is less than 1.18, the `kubernetes.io/ingress.class` annotation is added with the value of `ingress.className`. | `{}`                        |
+| `Capabilities.KubeVersion.GitVersion`       | The Kubernetes version running in the cluster. This is used to determine the API version of the Ingress resource to create.                                       |                             |
+| `gatewayd.fullname`                         | The full name of the deployment, used as the target of the Horizontal Pod Autoscaler.                                                                             |                             |
```

Comment:
Consider separating tables for each namespace/object (`xxxxx.something`), otherwise they look like duplicates.